src/components/clusterConfiguration/utils.ts

```diff
@@ -63,4 +66,6 @@ export const getInitialValues = (cluster: Cluster): ClusterConfigurationValues =
   sshPublicKey: cluster.sshPublicKey || '',
   isPullSecretEdit: !cluster.pullSecretSet, // toggles edit mode and drives validation
   hostSubnet: findMatchingSubnet(cluster.ingressVip, cluster.apiVip, getHostSubnets(cluster)),
+  useRedHatDnsService:
+    !!cluster.baseDnsDomain && managedDomains.map((d) => d.domain).includes(cluster.baseDnsDomain),
```

Comment:
What happens if I 
1. create a cluster
2. set my custom dns domain
3. save and close the form
4. open the form again
 ? 
Seems like `useRedHatDnsService` will be true, which is not correct ?

Reply:
ok, somehow I missed the includes :)